### DISCUSSION TITLE
using git and how exactly to have it installed for vs code

### DISCUSSION THREAD

**Discussion by MatthewProjects** [community/community#42170] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

<img width="428" alt="Screen Shot 2022-12-17 at 10 34 27 AM" src="https://user-images.githubusercontent.com/118500031/208555804-d2f2daf4-2759-4ece-a042-8aa2f42cb07e.png">

_can someone help me with these please_
 
vc code keeps wanting me to have git installed but im not sure which version I would need to use for it. why does it ask for a password before the installation of git???

<img width="442" alt="Screen Shot 2022-12-19 at 12 57 38 AM" src="https://user-images.githubusercontent.com/118500031/208555811-ea952dd4-dfaf-41d0-a3a0-d6a2a6039271.png">

**Comment by getlost01** [comment]
To use Git with VS Code, you will need to have Git installed on your computer. You can download the latest version of Git from the official Git website (https://git-scm.com/downloads). There are versions available for Windows, Mac, and Linux.

During the installation process, you may be asked to enter a password. This is typically required to grant the necessary permissions to install the software on your system. Depending on your operating system and the specific installation process, you may need to enter your user account password or provide other authentication credentials.

Once Git is installed, you can configure it to work with VS Code by following the instructions in the VS Code documentation (https://code.visualstudio.com/docs/editor/versioncontrol). This will typically involve specifying the path to the Git executable in your system and setting up any necessary authentication credentials.

If you are having trouble installing Git or getting it to work with VS Code, you may want to seek out additional resources or ask for help from a more experienced developer or refer to youtube.
